ts/Extensions/PatternFill.ts

```diff
@@ -300,23 +302,23 @@ SVGRenderer.prototype.addPattern = function (
     animation?: (boolean|Partial<AnimationOptions>)
 ): (SVGElement|undefined) {
     let pattern: (SVGElement|undefined),
-        animate = pick(animation, true),
-        animationOptions = animObject(animate),
         path: SVGAttributes,
+        attribs: SVGAttributes,
+        id = options.id;
```

Comment:
It is recommended to keep variables sorted in the ascending order as long as this is not causing any initialization problems (if you have something like `let x = this, a = x.a;` then it is going to stay as is).